modules/custom/social_advanced_queue/social_advanced_queue.install

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Install and update hooks for Social Swiftmailer.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions related to the installation of social_advanced_queue.
+ */
+function social_advanced_queue_install() {
+  // Get ultimate_cron.job.advancedqueue_cron settings.
+  $os_default_crons = [
+    'advancedqueue_cron',
+    'activity_send_email_cron',
```

Comment:
**_Just a suggestion (not necessary)_**
We can add a meaningful full title for these cron jobs as well.

or, alternatively,

Shall we also implement hook_cron_alter for providing a new title to the jobs, which may be more information to even admins instead of saying `Default cron handler`? 

See: ultimate_cron.api.php